src/Microsoft.AspNet.Razor.Runtime/XMLDocumentationProvider.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if !DNXCORE50 // Not used in coreclr
```

Comment:
Remove useless comment or describe what doesn't work (why this class isn't used with Core).